### PR TITLE
Move files to ui/ and update README

### PR DESCRIPTION
**Pull request by shreyaprabhu-g**

### CODE REVIEW

File: app.py
Comment (KonradSchieban) [inline app.py:1]: please move app.py into the ui folder as well.

File: requirements.txt
Comment (KonradSchieban) [inline requirements.txt:1]: please rebase to get the latest dependencies.

**Review by KonradSchieban — CHANGES_REQUESTED** [review]

File: requirements.txt
Comment (shreyaprabhu-g) [reply requirements.txt:1]: updated

**Review by KonradSchieban — APPROVED** [review]
LGTM

**Comment by KonradSchieban** [comment]
Thank you for working on the updates @shreyaprabhu-g ! 🥳

**Pull request merged**